build.sh

```diff
@@ -99,10 +99,10 @@ done
 # Clean if requested
 #
 if [ -n "$CLEAN" ]; then
-	b_log "Cleaning build dir"
+	b_log "Cleaning build dirs"
 	rm -rf "$PREFIX_BUILD" "$PREFIX_BUILD_HOST"
 	rm -rf "$PREFIX_FS"
-	#FIXME: this should also remove unpacked sources in 'ports' (the easiest option is to move them to _boot)
```

Comment:
Shouldn't this FIXME stay? I don't think it has been solved

Reply:
Solved, all ports are being unpacked to _build/$TARGET/...